pep-0581.rst

```diff
@@ -0,0 +1,310 @@
+PEP: 581
+Title: Using GitHub Issues for CPython
+Version: $Revision$
+Last-Modified: $Date$
+Author: Mariatta Wijaya <mariatta@python.org>
+Status: Draft
+Type: Process
+Content-Type: text/x-rst
+Created: 20-Jun-2018
+
+
+Abstract
+========
+
+This PEP outlines the steps required to migrate Python's issue tracker
+from Roundup to using GitHub Issues.
+
+
+Rationale
+=========
+
+CPython's development moved to GitHub on February 2017. All other projects within
+The PSF's organization are hosted on GitHub and are using GitHub issues.
+CPython is still using Roundup as the issue tracker in bugs.python.org (bpo).
+
+Why GitHub
+----------
+
+It has a lot of nice features readily available out of the box, that are not
+currently available in Roundup / bpo.
+
```

Comment:
I would start by saying that the form is much simpler, so it's easier to report a bug, no?